### ISSUE TITLE
download path defaults to "downloads" folder even if a custom path is set.

### ISSUE THREAD

**Issue by treos1** [Simon-Tesla/RaccoonyWebEx#18]
example: C:\Users\username here\Desktop\furry\raccoony\{siteName}\{author}\{submissionId}_{filename}_by_{author}.{extension}

even if i set the download path to something like this it simply creates a new folder in the "downloads" folder instead. so, instead of downloading the file to the folder on the desktop as intended, raccoony actually downloads to...

C:\Users\treos\Downloads\C_\Users\treos\Desktop\furry\raccoony\{siteName}\{author}\{submissionId}_{filename}_by_{author}.{extension}

so technically, it DOES let you set a custom download path but...instead of using the path you set, it creates it in the downloads folder instead.

and i know this worked properly back before they did the firefox quantum (or firefox chrome as i sometimes like to call it) update. so, i don't know what the problem could be but i felt it was worth reporting.

**Comment by Simon-Tesla** [comment]
Duplicate of #6 
tl;dr - this is a limitation of the WebExtension API introduced in Firefox Quantum. There are workarounds, but they're more difficult to implement. It's on my roadmap, but not in the immediate future.
